cypress/support/pages/settings.pageObject.js

```diff
@@ -0,0 +1,69 @@
+import PageObject from '../PageObject';
+
+class settingsPageObject extends PageObject {
+  url = '/settings';
+
+  loggedUser() {
+    cy.login();
+  }
+
+  updateUsername(username) {
+    cy.getByDataCy('settings-username')
+      .clear()
+      .type(username);
+  }
+
+  updateBio(bio) {
+    cy.getByDataCy('settings-bio')
+      .type(bio);
+  }
+
+  updateEmail(email) {
+    cy.getByDataCy('settings-email')
+      .clear()
+      .type(email);
+  }
+
+  updatePassword(password) {
+    cy.getByDataCy('settings-password')
+      .type(password);
+  }
+
+  submitButton() {
```

Comment:
```suggestion
  clickSubmitButton() {
```
the name should say what it does if it has any action